clients/java/src/main/java/io/camunda/zeebe/client/api/command/MigrateProcessInstanceCommandStep1.java

```diff
@@ -17,21 +17,23 @@
 
 import io.camunda.zeebe.client.api.ExperimentalApi;
 import io.camunda.zeebe.client.api.response.MigrateProcessInstanceResponse;
+import io.camunda.zeebe.gateway.protocol.GatewayOuterClass.MigrateProcessInstanceRequest;
```

Comment:
❓ This import is added only for the Javadoc, which would be a first in the java client's API. I see the benefit, and I don't think it has any negative impact but do you see any reason not to do this?